engine/item_effects.asm

```diff
@@ -2313,6 +2316,37 @@ ItemWasntUsedMessage:
 	jp PrintText
 ; f7e8
 
+Ball_ReplacePartyMonCaughtBall:
+	ld b, PARTYMENUACTION_00
+	call UseItem_SelectMon
+	jp c, ItemNotUsed_ExitMenu
+
+	ld a, [wCurItem]
+	ld b, a
+	ld a, [wCurPartyMon]
+	ld hl, wPartyMon1CaughtBall
+	call GetPartyLocation
+	ld a, [hl]
+	and CAUGHTBALL_MASK
+	cp b
+	
+	jp z, AlreadyInThatBallMessage
```

Comment:
`jr` is a more [optimized](https://github.com/pret/pokecrystal/wiki/Optimizing-assembly-code#relative-jumps) way to make short jumps. `AlreadyInThatBallMessage` is close enough to use `jr` instead of `jp`. Also, style-wise I'd expect the blank line to be after the jump, not before, since it separates the next "paragraph" of code which handles the swap.